Libraries/src/Amazon.Lambda.RuntimeSupport/Amazon.Lambda.RuntimeSupport.csproj

```diff
@@ -16,6 +16,13 @@
 	<LangVersion>9.0</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
This should be removable now that you got rid of the internal visible to and the strong naming is already being controlled via the `<repo-root>/buildtools/common.props`.

Reply:
removed.